soc/nordic_nrf/nrf54h/arm/Kconfig

```diff
@@ -3,32 +3,29 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-config SOC_NRF54H20
-	bool "nRF54H20"
-	depends on SOC_SERIES_NRF54HX
-
-if SOC_NRF54H20
-
-choice
-	prompt "nRF54H20 MCU Selection"
+config SOC_SERIES_NRF54HX
+	select ARM
+	select ARMV8_M_DSP
+	select CPU_CORTEX_M33
+	select HAS_NRFX
+	select HAS_NORDIC_DRIVERS
+	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
```

Comment:
I find this confusing, because the NRF54HX SoC series no longer can select ARM (or CPU_CORTEX_M33) in a multi-arch setup. These options should be selected by each CPU, even if it implies duplication (imagine eg we had a CORTEX-M33, a CORTEX-M0 and a VPR RISCV).

Reply:
Split it up properly (this would be easy to spot if there was a board supporting cpuppr)